apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -142,6 +142,24 @@
               </x-menu>
             </x-button>
           </x-buttons>
+          <x-buttons v-show="canManageTransactions && isManualCommit" class="">
```

Comment:
I would reorder these buttons so they look like this:

<img width="231" height="52" alt="Image" src="https://github.com/user-attachments/assets/38a797e0-ea53-4595-967e-691c24801de3" />

In this scenario, the primary button is `run`. So I think that it'll be more consistent to keep the `run` button on the right. There's a similar [article](https://www.nngroup.com/articles/ok-cancel-or-cancel-ok/) that talks about this if you are interested. Please feel free to disagree or add your own thoughts :)